org.virtualxt.VirtualXT.yml

```diff
@@ -0,0 +1,88 @@
+app-id: org.virtualxt.VirtualXT
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: virtualxt.freedos
+finish-args:
+  #- --share=network
+  - --share=ipc
+  #- --socket=wayland
+  #- --socket=fallback-x11
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
this is a very broad permission

Reply:
Well. The user might need to access to mounted media, serial and floppy devices in /dev
So I'm not sure if this can be narrowed anymore?

Reply:
This doesn't grant access to `/dev` anyway. Removable are `/media` or `/run/media`

Reply:
Would replacing `--filesystem=host` with `--filesystem=home --filesystem=/dev --filesystem=/media --filesystem=/run/media` be more appropriate then?

Reply:
`--device=all` should already be enough for the devices.